Cargo.toml

```diff
@@ -100,7 +100,7 @@ funty = "=1.1.0"
 
 # Pin, because dialoguer pulls it in, but 1.4.x and newer has MSRV 1.51.0. With
 # the pin here, we enforce the build to not use 1.4.0 or newer.
```

Comment:
We probably don't need this version constraint anymore (our current MSRV is higher) but this needs to be checked and Cargo.toml updated accordingly.